lib/services/auth-service.js

```diff
@@ -37,23 +39,12 @@ function authServiceFactory(
 
     AuthService.prototype.init = function(){
         var self = this;
-        this.strategies = [];
 
         self.algorithm = 'HS256';
         self.secretOrKey = '';
         self.ignoreTokenExpiration = false;
         self.encoder = 'base64';
 
-        self.hashConfig = {
```

Comment:
The on-core PR was updated back to using pbkdf2 over bcrypt.  The settings below are copied over into the on-core change.